VoteCounter/src/GUI.cs

```diff
@@ -27,10 +30,55 @@ private void btnGenerateVoteCount_Click(object sender, EventArgs e)
 
             if (Int32.TryParse(txtStartingPost.Text, out startingPostNumber) && Int32.TryParse(txtEndingPost.Text, out endingPostNumber))
             {
+                //make a List
```

Comment:
That's exactly what you do. It's just to prevent the btnGenerateVoteCount_Click() function getting too long